test/example_kb_sdk_app_server_test.py

```diff
@@ -6,8 +6,7 @@
 
 from example_kb_sdk_app.example_kb_sdk_appImpl import example_kb_sdk_app
 from example_kb_sdk_app.example_kb_sdk_appServer import MethodContext
-from example_kb_sdk_app.authclient import KBaseAuth as _KBaseAuth
-
+from installed_clients.authclient import KBaseAuth as _KBaseAuth
```

Comment:
The auth client doesn't work anymore with later versions of kb-sdk inside of the kb-sdk image..